lib/android_rename_steps.dart

```diff
@@ -24,7 +24,7 @@ class AndroidRenameSteps {
     }
     String? contents = await readFileAsString(PATH_BUILD_GRADLE);
 
-    var reg = RegExp('applicationId "(.*)"', caseSensitive: true, multiLine: false);
+    var reg = RegExp('applicationId = "(.*)"', caseSensitive: true, multiLine: false);
```

Comment:
It'll break compatibility with old versions. 

use this regex instead `applicationId\s*=?\s*"(.*)"`